docs/spec_draft.md

```diff
@@ -582,6 +583,78 @@ Produces a `result` tensor from a constant `value`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.convert
+
+### Semantics
+
+Performs an element-wise conversion of values from one element type to another
+on `operand` tensor and produces a `result` tensor.
+
+More formally, conversion from element `A` of `operand` to element `B` of
```

Comment:
Also can we have a table like the following to detail all the conversions

| `operand` type, `T` | `result` type `T'` | `B` |
|--|--|--|

and merging multiple cells related to column `result` type like https://jsfiddle.net/7h89y55r/